extensions/functions/src/main/java/io/crate/operation/aggregation/HyperLogLogDistinctAggregation.java

```diff
@@ -245,7 +244,10 @@ public DocValueAggregator<?> getDocValueAggregator(LuceneReferenceResolver refer
                     @Override
                     public void apply(RamAccounting ramAccounting, int doc, HllState state) throws IOException {
                         if (super.values.advanceExact(doc) && super.values.docValueCount() == 1) {
-                            BytesRef ref = super.values.nextValue();
+                            // TODO: for small cardinality fields it might be worth hashing all terms
```

Comment:
Can you remove these TODOs and instead create an issue?

Reply:
I opened https://github.com/crate/crate/issues/16439